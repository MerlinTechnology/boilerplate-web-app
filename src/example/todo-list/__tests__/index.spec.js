import React from 'react'
import renderer from 'react-test-renderer'
import thunk from 'redux-thunk'
import { routerMiddleware } from 'react-router-redux'
import { browserHistory } from 'react-router'
import configureMockStore from 'redux-mock-store'

import TodoListWrapper from '../'
import { initialState } from '../redux'

const middleware = [thunk, routerMiddleware(browserHistory)]
const mockStore = configureMockStore(middleware)

it('renders correctly', () => {
    const store = mockStore({ todoList: initialState })
    const component = renderer.create(
        <TodoListWrapper store={store}/>
    )

    expect(component).toMatchSnapshot()
})
