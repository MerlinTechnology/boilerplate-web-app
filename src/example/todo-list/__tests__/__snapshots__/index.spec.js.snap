// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListWrapper renders correctly 1`] = `
<div
  className="main-wrapper"
>
  <div
    className="main-container row"
  >
    <div
      className="col"
    >
      <div>
        <input
          autoFocus="true"
          onKeyPress={[Function]}
          placeholder="What to do?"
          type="text"
        />
      </div>
      <ul
        className="container"
      />
    </div>
  </div>
</div>
`;
